connections/src/app/features-michael/profile-loader/profile-loader.component.html

```diff
@@ -22,40 +22,64 @@ <h1 class="text-lg font-bold text-center mb-3"><span>PROFILE LOADER</span></h1>
                 </button>
             </div>
             <mat-accordion>
-            <div *ngIf="bio != ''">
-                <mat-expansion-panel hideToggle>
-                    <mat-expansion-panel-header>
-                        <mat-panel-title>Bio</mat-panel-title>
-                    </mat-expansion-panel-header>
-                    <p>Bio: {{bio}}</p>
-                </mat-expansion-panel>
-                
-            </div>
-            <div *ngIf="linksS.length > 0">
-                <mat-expansion-panel hideToggle>
-                    <mat-expansion-panel-header>
-                        <mat-panel-title>S Links</mat-panel-title>
-                    </mat-expansion-panel-header>
-                    <li *ngFor="let link of linksS | slice:0:4" ><img class="image" [src]="showImage(link.url_label)">{{ link.link_type }}
-                        , {{link.url_link}}</li>
+                <div *ngIf="bio != ''">
+                    <mat-expansion-panel hideToggle>
+                        <mat-expansion-panel-header>
+                            <mat-panel-title>Bio</mat-panel-title>
+                        </mat-expansion-panel-header>
+                        <p>Bio: {{bio}}</p>
+                    </mat-expansion-panel>
+
+                </div>
+                <div *ngIf="linksS.length > 0">
+                    <mat-expansion-panel hideToggle>
+                        <mat-expansion-panel-header>
+                            <mat-panel-title>S Links</mat-panel-title>
+                        </mat-expansion-panel-header>
+                        <li *ngFor="let link of linksS | slice:0:4"><img class="image"
+                                [src]="showImage(link.url_label)">{{ link.link_type }}
+                            , {{link.url_link}}</li>
                         <div *ngIf="linksS.length > 4">
                             <p>+ {{linksSAdjusted}} more</p>
                         </div>
-                </mat-expansion-panel>
```

Comment:
Ya. I don't like seeing the expansion panel here in profile. So that can be gotten rid of. It was fine over in the Connections CRUD app, but not here.
lets follow close to figma